backend/src/main/java/com/example/user/UserService.java

```diff
@@ -25,6 +28,25 @@ public Optional<User> findUserByName(String name) {
         return userRepository.findByName(name);
     }
 
+    public int getNumberOfUsersOnline() {
+        List<User> users = userRepository.findAll();
+        List<String> ips = new ArrayList<String>();
+        for (User user: users) {
+            try {
+                if (InetAddress.getByName(user.getNetbar_ip()).isReachable(500)) {
```

Comment:
O ile dobrze to rozumiem, to w ten sposób sprawdzasz czy użytkownik, który logował się do serwera gry ma włączony komputer, a nie to czy jest online w grze